integration-tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -142,7 +191,7 @@ exports[`works with node assert 1`] = `
 
   ● assert
 
-    assert.equal(received, expected) or assert(received) 
+    thrown: \\"assert.equal(received, expected) or assert(received) 
```

Comment:
this shouldn't change, it's `AssertionError` from node (https://nodejs.org/api/errors.html#errors_class_assertionerror). Seems like something is a bit off about the check?

Reply:
It turns out those are just strings, and also that Errors never make it to the message formatter.